functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
+            if n[j] == t - n[i]:
+                return [i, j]
```

Comment:
return something if there are no two items in n that add up to t

Reply:
see commit [cf6ef08](https://github.com/nikkitorab/more-code-reviews/pull/1/commits/cf6ef08fb0faa4ffad5401cb8a27866e9e699f0c)